OnionBrowser/Browsing/BrowsingViewController+Tabs.swift

```diff
@@ -72,8 +72,8 @@ UICollectionViewDropDelegate, TabCellDelegate {
 				let tab = tabs[indexPath.row]
 
 				cell.title.text = tab.title
-
-				tab.add(to: cell.container)
```

Comment:
This has a counterpart which should be removed then, too.

Reply:
I didn't catch it( Sorry please clarify do you mean.